go/cmd/remotetool/main.go

```diff
@@ -29,35 +36,28 @@ type OpType string
 const (
 	downloadActionResult OpType = "download_action_result"
 	showAction           OpType = "show_action"
+	downloadBlob         OpType = "download_blob"
+	downloadMerkleTree   OpType = "download_merkle_tree"
```

Comment:
probably name this download_tree / download_dir ?

Reply:
Done.